advanced/creating-a-skin.md

```diff
@@ -4,48 +4,56 @@ title: Create a skin for TinyMCE
 title_nav: Create a skin
 description_short: Introducing skin creation.
 description: Introducing skin creation, less and icon modification.
-keywords: create creator skin icomoon
+keywords: create creator skin icon
 ---
 
-Create a new skin to alter the appearance of TinyMCE. The simplest way to create a new skin is to use the [TinyMCE Skin Creator](http://skin.tinymce.com/) tool. The other option is to manually modify `less` files and build them using the `Node.js` build process. The tutorial below focuses on the manual method of skin creation.
+Create a new skin to alter the appearance of TinyMCE.
+
+This guide assumes you have basic understanding of [Node](http://nodejs.org) and [Gulp](http://gulpjs.com) and know your way around a terminal.
+
+> Note: The  [TinyMCE Skin Creator](http://skin.tinymce.com/) only supports TinyMCE 4.
 
 ## Preparation
 
-Download the [TinyMCE development package](https://www.tinymce.com/download/) to manually build the `less` files. This package contains the source code for the entire TinyMCE project and the `less` files needed to skin the editor or use the [GitHub version](https://github.com/tinymce/tinymce/).
+Skin for TinyMCE 5 are written in [less](http://lesscss.org), a popular CSS preprocessor. Making a skin primarily involves modifying variables which you can consider a kind of API. You are not supposed to modify or override CSS rules, instead you override the less variables in your skin files. This makes updating TinyMCE much easier.
 
-## Building skin.min.css from less files
+1. Download the [**LINK MISSING** TinyMCE skin tools](#).
+2. Install all dependencies with `npm install`.
+3. Issue the `gulp` command which also spins up a webserver for previewing the files (If you just want to build the files, use `gulp build`) .
```

Comment:
it should come down with `npm install`, but that means you need to use `npm run build` rather than `gulp build` since it might not be on the global path